### PR TITLE
Em 3615 s2s implementation

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-3615

### Change description ###
Added in S2S config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java [hunk 1]
```diff
@@ -36,4 +38,6 @@ public HrsHttpClient provideHrsHttpClient(final Retrofit retrofit) {
         return retrofit.create(HrsHttpClient.class);
     }
 
+    @Autowired
+    private AuthTokenGenerator authTokenGenerator;
```
Comment (benouaer) [inline src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java:42]: why autowire it here?

**Comment by benouaer** [comment]
can't seen where you're generating the token?

File: src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java (hunk 1 above)
Comment (yogesh-hullatti) [reply src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java:42]: So that it will be ready along with other beans as part of Config

**Comment by yogesh-hullatti** [comment]
> can't seen where you're generating the token?

Token will be generated as part of teh code that will be consuming the AuthTokenGenerator. 
i.e. At the point where we will have the logic to call the em-hrs-api.

File: src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java (hunk 1 above)
Comment (benouaer) [reply src/main/java/uk/gov/hmcts/reform/em/hrs/ingestor/config/AppConfig.java:42]: but it's already added as a bean in the other config file that you created

**Review by benouaer — APPROVED** [review]

**Pull request merged**